<ui version="4.0" >
 <class>ImportEISOMSPage2</class>
 <widget class="QWidget" name="ImportEISOMSPage2" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>4</number>
   </property>
   <property name="topMargin" >
    <number>4</number>
   </property>
   <property name="rightMargin" >
    <number>4</number>
   </property>
   <property name="bottomMargin" >
    <number>4</number>
   </property>
   <property name="horizontalSpacing" >
    <number>4</number>
   </property>
   <property name="verticalSpacing" >
    <number>4</number>
   </property>
   <item row="0" column="0" colspan="4" >
    <widget class="CProgressBar" name="progressBar" >
     <property name="value" >
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="lblAcceptedLabel" >
     <property name="text" >
      <string>принято</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLabel" name="lblAccepted" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string>0</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::PlainText</enum>
     </property>
     <property name="scaledContents" >
      <bool>false</bool>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QLabel" name="lblIntegrityErrorLabel" >
     <property name="text" >
      <string>нет в реестре</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QLabel" name="lblIntegrityError" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string>0</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::PlainText</enum>
     </property>
     <property name="scaledContents" >
      <bool>false</bool>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="lblRefusedLabel" >
     <property name="text" >
      <string>отказано</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLabel" name="lblRefused" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string>0</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::PlainText</enum>
     </property>
     <property name="scaledContents" >
      <bool>false</bool>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QLabel" name="lblChangeDisabledLabel" >
     <property name="text" >
      <string>не подлежит изменению</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="QLabel" name="lblChangeDisabled" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string>0</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::PlainText</enum>
     </property>
     <property name="scaledContents" >
      <bool>false</bool>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>151</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2" >
    <widget class="QLabel" name="lblUncheckedLabel" >
     <property name="text" >
      <string>не обработано ЕИС-ОМС</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" >
    <widget class="QLabel" name="lblUnchecked" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string>0</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::PlainText</enum>
     </property>
     <property name="scaledContents" >
      <bool>false</bool>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>392</width>
       <height>111</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CProgressBar</class>
   <extends>QProgressBar</extends>
   <header>library/ProgressBar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
